client/my-sites/checkout/checkout/redirect-payment-box.jsx

```diff
@@ -197,21 +204,41 @@ export class RedirectPaymentBox extends PureComponent {
 	};
 
 	renderButtonText() {
+		const testSealsCopy = 'variant' === abtest( 'checkoutSealsCopyBundle' ),
+			renewalText = testSealsCopy
+				? this.props.translate( 'Complete subscription purchase with %(paymentProvider)s', {
+						args: {
+							paymentProvider: this.getPaymentProviderName(),
+						},
+						context: 'Pay button on /checkout',
+				  } )
+				: this.props.translate( 'Purchase %(price)s subscription with %(paymentProvider)s', {
+						args: {
+							price: this.props.cart.total_cost_display,
+							paymentProvider: this.getPaymentProviderName(),
+						},
+						context: 'Pay button on /checkout',
+				  } ),
+			purchaseText = testSealsCopy
+				? this.props.translate( 'Complete purchase with %(paymentProvider)s', {
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?